homeassistant/components/weather/__init__.py

```diff
@@ -221,6 +222,28 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
         ],
         supports_response=SupportsResponse.ONLY,
     )
+    component.async_register_entity_service(
+        SERVICE_GET_FORECAST_VALUE,
+        {
+            vol.Required("attribute"): vol.In(
+                (
+                    "temperature",
+                    "templow",
+                    "precipitation",
+                    "precipitation_probability",
+                    "wind_speed",
+                    "humidty",
```

Comment:
Wrong spelling. Please don't use magic strings, but ATTR_WEATHER-constants instead to avoid that.

Reply:
I guess we should all of the float ones, as example `wind_gust_speed` would be missing